### PR TITLE
fix: minor correction to docs in FMU based workflow section

### PR DESCRIPTION
**Pull request by abhilash-kumar-nair**

### CODE REVIEW

File: docs/source/modelon.impact.client.experiment_definition.rst [hunk 1]
```diff
@@ -52,6 +52,14 @@ modelon.impact.client.experiment\_definition.model\_based module
...
    :show-inheritance:
 
+modelon.impact.client.experiment\_definition.modifiers module
+-------------------------------------------------------------
+
+.. automodule:: modelon.impact.client.experiment_definition.modifiers
+   :members:
+   :undoc-members:
+   :show-inheritance:
+
```
Comment (iakovn) [inline docs/source/modelon.impact.client.experiment_definition.rst:62]: I don't think we should expose this module in public docs yet. When we do we should document the classes/methods.

**Review by iakovn — CHANGES_REQUESTED** [review]
Minor corrections are fine 💯 but please don't include the new module ⛔

File: docs/source/modelon.impact.client.experiment_definition.rst (hunk 1 above)
Comment (abhilash-kumar-nair) [reply docs/source/modelon.impact.client.experiment_definition.rst]: Good catch! :100:

Comment (abhilash-kumar-nair) [reply docs/source/modelon.impact.client.experiment_definition.rst]: Thanks for the review @iakovn ! I removed the docs addition. Could you please take another look?

**Review by iakovn — APPROVED** [review]
LGTM! 👍

**Pull request merged**